Source/Core/DolphinQt/Config/GameConfigWidget.cpp

```diff
@@ -196,207 +192,171 @@ void GameConfigWidget::CreateWidgets()
+  gfx_layout->addWidget(gfx_tabs);
+
+  connect(tab_widget, &QTabWidget::currentChanged, this, [this, editor_index](int index) {
+    // Update the ini editor after editing other tabs.
+    if (index == editor_index)
+    {
+      // Layer only auto-saves when it is destroyed.
+      m_layer->Save();
+
+      // There can be multiple ini loaded for a game, only replace the one related to the game
+      // ini being edited.
+      for (int i = 0; i < m_local_tab->count(); i++)
+      {
+        if (m_local_tab->tabText(i).toStdString() == m_game_id + ".ini")
+        {
+          m_local_tab->removeTab(i);
+
+          auto* edit = new GameConfigEdit(
+              nullptr,
+              QString::fromStdString(File::GetUserPath(D_GAMESETTINGS_IDX) + m_game_id + ".ini"),
+              false);
 
-void GameConfigWidget::ConnectWidgets()
-{
-  // Buttons
-  connect(m_refresh_config, &QPushButton::clicked, this, &GameConfigWidget::LoadSettings);
-
-  for (QCheckBox* box :
-       {m_enable_dual_core, m_enable_mmu, m_enable_fprf, m_sync_gpu, m_emulate_disc_speed,
-        m_use_dsp_hle, m_manual_texture_sampling, m_use_monoscopic_shadows})
-    connect(box, &QCheckBox::stateChanged, this, &GameConfigWidget::SaveSettings);
-
-  connect(m_deterministic_dual_core, &QComboBox::currentIndexChanged, this,
-          &GameConfigWidget::SaveSettings);
-  connect(m_depth_slider, &QSlider::valueChanged, this, &GameConfigWidget::SaveSettings);
-  connect(m_convergence_spin, &QSpinBox::valueChanged, this, &GameConfigWidget::SaveSettings);
-}
+          m_local_tab->insertTab(i, edit, QString::fromStdString(m_game_id + ".ini"));
+          break;
+        }
+      }
+    }
 
-void GameConfigWidget::LoadCheckBox(QCheckBox* checkbox, const std::string& section,
-                                    const std::string& key, bool reverse)
-{
-  bool checked;
-  if (m_gameini_local.GetOrCreateSection(section)->Get(key, &checked))
-    return checkbox->setCheckState(checked ^ reverse ? Qt::Checked : Qt::Unchecked);
+    // Update other tabs after using ini editor.
+    if (m_prev_tab_index == editor_index)
+    {
+      // Load won't clear deleted keys, so everything is wiped before loading.
+      m_layer->DeleteAllKeys();
+      m_layer->Load();
+      Config::OnConfigChanged();
+    }
 
-  if (m_gameini_default.GetOrCreateSection(section)->Get(key, &checked))
-    return checkbox->setCheckState(checked ^ reverse ? Qt::Checked : Qt::Unchecked);
-  checkbox->setCheckState(Qt::PartiallyChecked);
+    m_prev_tab_index = index;
+  });
+
+  auto help_icon = style()->standardIcon(QStyle::SP_MessageBoxQuestion);
+  auto* help_label = new QLabel(tr("These settings override core Dolphin settings."));
+  help_label->setToolTip(
+      tr("Italics mark a default game setting, bold marks user "
+         "settings.\nRight-click to remove user settings.\n AA settings are disabled when the "
+         "global backend doesn't match the game setting."));
```

Comment:
* Either "a default game setting" should be "default game settings" or "user settings" should be "a user setting", for consistency.
* I assume the space before AA is a mistake.
* What AA means might not be clear to the user (or to translators, for that matter). I would suggest writing out the full word.
* Maybe "graphics backend" instead of "backend"? We do have other backend options in Dolphin too, even though I suppose they're not visible in the game properties dialog right now.

By the way, after reading this text and going over to the Graphics tab, I was surprised to find that nothing was marked in italics there. From reading the code, it looks like italics only apply to the General tab. Wouldn't it make more sense for italics to work the same way in the Graphics tab as in the General tab?

Reply:
>By the way, after reading this text and going over to the Graphics tab, I was surprised to find that nothing was marked in italics there. From reading the code, it looks like italics only apply to the General tab. Wouldn't it make more sense for italics to work the same way in the Graphics tab as in the General tab?

Yes, but the system-based game settings won't be loaded for the other tabs.  I think it would take another PR with some clever logic.  Since it only needs to italicize once on loading, you could maybe iterate through gameconfigwindow child widgets/options because I expose the Config Location of each option.